src/apps/code-editor/src/app/components/FileActions/FileActions.less

```diff
@@ -25,6 +25,9 @@
 
       display: flex;
       align-items: center;
+      button {
```

Comment:
@d88naimi I see you did this to apply a margin on the design-system bound CopyButton. I would like to see this case being solved in an alternative fashion to ease its eventual transition to 'material'. Adding this here makes it untransparent and bug-prone when we swap it out to 'material' because this style can easily bleed into it.

One alt solution that comes to mind is to wrap the CopyButton in the wrapper and applying the margin on there. This will give us more transparency of what is being applied and why as well as avoiding the style bleed we have encountered with styles selecting native elements.